app/models/favorite.rb

```diff
@@ -0,0 +1,8 @@
+class Favorite < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :tattoo
+
+  validates :user_id, presence: true
+  validates :tattoo_id, presence: true
+  validates :user_id, uniqueness: { scope: :tattoo_id }
```

Comment:
Chop off the `_id`s from all of these! It should be `validates :user, presence: true` etc.